metaparser/mp.lua

```diff
@@ -1290,15 +1300,15 @@ function mp:match(verb, w, compl)
 	local unknown = {}
 	local multi = {}
 	local vargs
+	local a = {}
+  for k, v in ipairs(w) do
+    if k < verb.verb_nr or k >= verb.verb_nr + verb.verb_len then
+      table.insert(a, v)
+    end
+  end
```

Comment:
We could not remove this from loop. Because "a" is modified inside loop. This code resets "a" state.

Reply:
we SHOULD remove this because the setting code does not use any variables from the loop. If we reset 

We can calculate the array before the loop, and just reset this `a` to calculated state instead of calculating the step all over again.